js/src/forum/addComposerAutocomplete.js

```diff
@@ -116,7 +116,7 @@ export default function addComposerAutocomplete() {
                 if (!userMatches(user)) return;
 
                 suggestions.push(
-                  makeSuggestion(user, '@' + user.username(), '', 'MentionsDropdown-user')
+                  makeSuggestion(user, `@"${user.displayName().replace(/"#[a-z]{0,3}[0-9]+/, '_')}"#${user.id()}`, '', 'MentionsDropdown-user')
```

Comment:
This regex is repeated. Can we move it out somewhere? maybe `const cleanDisplayName = user => ....`, and use that instead?